python/lsst/ts/ess/earthquake/q330_utils.py

```diff
@@ -34,6 +36,117 @@
 
 import ctypes
 import enum
+from dataclasses import dataclass
+
+
+class ChannelName(enum.Enum):
+    """Enum of all channel names.
+
+    See https://ds.iris.edu/ds/nodes/dmc/data/formats/seed-channel-naming/ and
+    https://scedc.caltech.edu/data/station/seed.html for more information.
+    """
+
+    """Broad band, high gain, east."""
+    BHE = enum.auto()
```

Comment:
I would really be much happier if you used more expressive names for these enums instead of shortening them. 

You could do 

```
BROAD_BAND_HIGH_GAIN_EAST = enum.auto()
BROAD_BAND_HIGH_GAIN_NORTH = enum.auto()
...
```

And then you wouldn't need to add the docstring.

Reply:
I realized that I really don't need the enum so I removed it.